### PR TITLE
bindata/bootkube/bootstrap-manifests: add etcdctl container to bootstrap-etcd

### PR DESCRIPTION
**Pull request by hexfusion**
This PR adds the etcdctl container to the bootstrap node etcd pod adding to the debugability of bootstrap failures.

### CODE REVIEW

**Comment by hexfusion** [comment]
cc @hasbro17

File: bindata/bootkube/bootstrap-manifests/etcd-member-pod.yaml [hunk 1]
```diff
@@ -39,8 +63,6 @@ spec:
     securityContext:
       privileged: true
     volumeMounts:
-    - name: discovery
-      mountPath: /run/etcd/
```
Comment (hexfusion) [inline bindata/bootkube/bootstrap-manifests/etcd-member-pod.yaml:43]: this is old 4.3 cruft FTR

**Review by hasbro17 — APPROVED** [review]
/lgtm

**Comment by hexfusion** [comment]
sorry had one more cleanup on mounts

**Comment by hexfusion** [comment]
https://github.com/openshift/cluster-etcd-operator/blob/3b7350c5a90342b25045d39e62cd2e813695eebd/bindata/bootkube/bootstrap-manifests/etcd-member-pod.yaml#L72

sigh, missed one more..

**Comment by hasbro17** [comment]
/lgtm

Third time's the charm.

**Comment by hexfusion** [comment]
unrelated

/retest-required

**Pull request merged**